api4/config.go

```diff
@@ -35,6 +35,11 @@ func configReload(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if *c.App.Config().ExperimentalSettings.RestrictSystemAdmin {
+		c.Err = model.NewAppError("addLicense", "api.restricted_system_admin", nil, "", http.StatusBadRequest)
```

Comment:
Wrong function name in the error, should be `configReload`instead of `addLicense`

Reply:
Gah! I thought I had fixed all of these. Thanks :)